extensions/wikia/ListGlobalUsers/DatabaseGlobalUsersService.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+class DatabaseGlobalUsersService implements GlobalUsersService {
+	public function getGroupMembers( array $groupSet ): array {
+		if ( empty( $groupSet ) ) {
+			return [];
+		}
+
+		global $wgExternalSharedDB;
+		$dbr = wfGetDB( DB_SLAVE, [], $wgExternalSharedDB );
```

Comment:
I'm not aware of MW conventions, but wouldn't it be better to pass the db handle in the constructor, or at least as an argument to this function?